src/main.rs

```diff
@@ -93,7 +95,10 @@ async fn main() {
         port as u16,
     );
 
-    let app = Router::new()
+    let mut headers = HeaderMap::new();
+    headers.insert("x-reduct-api", HeaderValue::from_static(version));
```

Comment:
I also think that it is better to use only <major_version>.<minor_version> for the API version, because we don't change it in patch version.